www/src/utils/getParagonComponentsTypes.js

```diff
@@ -8,6 +8,9 @@ const getParagonComponentsTypes = (components) => {
     const isContext = !!component.Consumer && !!component.Provider;
     let componentType;
     switch (true) {
+      case componentName.toLowerCase() === 'paragon':
+        componentType = 'Paragon';
+        break;
```

Comment:
Interesting; didn't know we had a usage of `import * as paragon from '@edx/paragon';`!